instrumentation/afl-compiler-rt.o.c

```diff
@@ -793,6 +810,7 @@ static void __afl_start_forkserver(void) {
         //(void)nice(-20);
 
         signal(SIGCHLD, old_sigchld_handler);
+        signal(SIGTERM, old_sigterm_handler);
```

Comment:
this is something I am not happy with. an extra signal handler change after every fork. but I have no clue what a better solution could be. ideas?

Reply:
If there was no way to avoid signal(SIGCHLD) handling here, I doubt there is one for signal(SIGTERM) handling...